src/WatchDogs.Infrastructure.FakeSource/Watcher.cs

```diff
@@ -16,13 +17,15 @@ public class Watcher : IWatcher
     private readonly CancellationTokenSource _cts = new();
     private readonly IFakeTradeGenerator _dataGenerator;
     private readonly ILogger _logger;
```

Comment:
You forgot to inject the logger

Reply:
Had to do little workaround because serilog was producing problems